packages/dcos-integration-test/extra/test_meta.py

```diff
@@ -0,0 +1,82 @@
+"""
```

Comment:
The core of this has served us well in https://github.com/mesosphere/dcos-e2e where we use a similar pattern.